scipy-1.0/paper.tex

```diff
@@ -902,6 +902,49 @@ \subsection*{Maintainers and contributors}
 are selected on a rolling basis from contributors with a significant history of
 high-quality contributions.
 
+\subsection*{Funding}
+
+
+
+%100,000 annual salary, 112 person-years, 414,473 lines of code -> 11,192,622 (https://www.openhub.net/p/scipy/estimated\_cost, which uses the Basic COCOMO model).
+
+The development cost of SciPy is estimated in excess of 10M dollars by
+Open Hub\cite{SciPyValue}.  Yet, the project is largely unfunded, having been
+developed predominantly by graduate students, faculty, and members of industry
+in their free time. Small amounts of funding have been applied with success:
+some meetings were sponsored by universities and industry, Google's Summer of Code
+program supported infrastructure and algorithmic work, and teaching grant funds were
+used early on to develop documentation.  However, these small amounts of
+sporadic funding are not commensurate with the enormous stack of important scientific
+software, like those that are used to plan national space missions, that rely
+on SciPy.
+With traditional grant mechanisms, it is easier to buy software licenses than to support the diverse development effort needed to develop open source software—a challenge that we hope to see national funding agencies address.
+The scientific Python stack is also playing an increasingly important role in industry, and while there are some encouraging signals such as TideFlow, that sector still spends comparatively little on community-developed software.
+For open source projects to remain a healthy underpinning infrastructure for international scientific and industrial endeavors, it needs more diverse spending to support planning, development, management, and infrastructure.
```

Comment:
to avoid repeating "infrastructure", perhaps use "underpinning" as a noun rather than adjectival noun.
```suggestion
For open source projects to remain a healthy underpinning for international scientific and industrial endeavors, it needs more diverse spending to support planning, development, management, and infrastructure.
```